### PR TITLE
[SEATUNNEL#1060] Enable the MagicNumber rule in checkstyle and fix them

### LINKED ISSUE #1060: [Feature][CheckStyle] Enable the MagicNumber rule in checkstyle and fix them
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Enable the MagicNumber rule in checkstyle and fix them

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by asdf2014**
## Purpose of this pull request

Fixes #1060

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-connectors/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisOutputFormat.java [hunk 1]
```diff
@@ -44,6 +44,7 @@
  **/
 public class DorisOutputFormat<T> extends RichOutputFormat<T> {
     private static final Logger LOGGER = LoggerFactory.getLogger(DorisSinkFunction.class);
+    private static final long DEFAULT_INTERVAL_MS = TimeUnit.MINUTES.toMillis(1);
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```
Comment (CalvinKirs) [inline seatunnel-connectors/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisOutputFormat.java]: Why expand this value?

Comment (asdf2014) [reply seatunnel-connectors/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisOutputFormat.java]: You are right, it should be `TimeUnit.SECONDS.toMillis(1)`, let me fix it

Comment (asdf2014) [reply seatunnel-connectors/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisOutputFormat.java]: Done

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**